### PR TITLE
[13] Style the Product Details Page using CSS

### PR DESCRIPTION
**Pull request by Egwolo2**
prod1
prod2
popad1
poipad2
pophone1
pophone2
pophone3
phone4

## Task: `Style the Product detailes page using CSS`
- `Created and styled the product page with css
Added some pictures to the image file
created a css fill for product page
changed the HTML product name from index to product.`

### CODE REVIEW

File: submissions/egwoloOkorodudu/HTML/product.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
```
Comment (ToluAkin) [inline submissions/egwoloOkorodudu/HTML/product.html:1]: You need to style the already created product page. You are not meant to create a new file for product page.@Egwolo2

**Review by ToluAkin — CHANGES_REQUESTED** [review]
Nice comment details in the css. Well done!

**Comment by ToluAkin** [comment]
You need to rename your images such that there are no space between them. @Egwolo2

File: submissions/egwoloOkorodudu/HTML/product.html (hunk 1 above)
Comment (Egwolo2) [reply submissions/egwoloOkorodudu/HTML/product.html:1]: I think that was what i did.  I only changed the name from index.html to product.html

Comment (adaugochi) [reply submissions/egwoloOkorodudu/HTML/product.html:1]: ok

**Review by adaugochi — APPROVED** [review]

**Pull request merged**